source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -23,20 +23,81 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
 
-## See Also
+**Note:** Currently it should be installed on a new machine, separate from the engine. 
+It can be installed on a dedicated VM. 
 
-[oVirt Metrics – EFK](https://sradcoblog.wordpress.com/2016/07/19/ovirt-metrics-elk/)
+Please follow one of these options. Adding SSO is optional:
+ 
+  * [Metrics Store setup on top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
+
+  OR
+
+  * [Metrics Store setup on top of OpenShift without oVirt Engine SSO](https://github.com/ViaQ/Main/blob/master/README-mux.md)
+
+[![oVirt Metrics data flow](/images/wiki/oVirtMetricsDataFlow.jpg)](images/wiki/oVirtMetricsDataFlow.jpg)
+
+Once you have finished this step, you should have:
+
+  * Kibana - <https://kibana.{hostname}>
+  * Openshit portal - <https://openshift.{hostname}>
+
+
+**oVirt Hypervisors and Engine Setup -**
+
+oVirt machines on versions 4.1.1 and above include Fluentd and Collectd packages.
+Now we need to deploy and configure Collectd and Fluentd to send the data to the central Metrics Store:
+
+1. Install / Upgrade and setup oVirt Engine 4.1, latest release.
+
+2. Install / Upgrade and activate one or more hosts, 4.1 latest release.
+
+3. Copy the CA certificate created earlier on [the metrics store machine](https://github.com/ViaQ/Main/blob/master/README-mux.md#getting-the-shared_key-and-ca-cert), to the engine machine.
+
+
+   On the metrics store machine, run:
+
+        # scp /path/to/ca/certificate.cert root@<fqdn/of/engine/machine>:/etc/ovirt-engine-metrics
+
+4. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
+
+   On the oVirt engine machine, run:
+
+        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine/config.yml
+
+5. Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:
+
+        # vi /etc/ovirt-engine-metrics/config.yml.example
+
+    Replace the example values with your specific environment details:
+     
+     * "fluentd-server.example.com" - The fully qualified domain name of the metrics store machine
+     
+     * "my_shared_key" - The shared key configured in Fluentd on the metrics store machine
+     
+     * "/path/to/fluentd_ca_cert.pem" - The path to the Fluentd CA certificate
+     * "ovirt_env_name" - The Environment name. Can be used to identify data collected in a single central store sent from more than one oVirt engine.
+
+6. On the engine machine, run as root:
+
+        # /usr/share/ovirt-engine-metrics/setup/ansible/configure_ovirt_hosts_for_metrics.sh
+
+It runs the Ansible script that configures Collectd and Fluentd on the oVirt engine and hypervisors.
+
+It should finish without errors.
+
+When finished, you should be able to see statistics in the Kibana console, at the address you configured previously on the [metrics store machine](https://github.com/ViaQ/Main/blob/master/README-mux.md#running-kibana), about your hosts, VMs, etc.
```

Comment:
Suggested change:
Once finished, you can view host, VM and other statistics in the Kibhana console, at the address configured earlier on in this procedure (see metrics store machine). 

Comment: "metrics store machine" is the link.